### ISSUE TITLE
TextInput placeholder triggers wrong contentSize.height of onContentSizeChange

### ISSUE THREAD

**Issue by kevinscience** [facebook/react-native#19076]
```
<TextInput
                    placeholder={'12345678'}
                    multiline={true}
                    onContentSizeChange={(e) => console.log(e.nativeEvent.contentSize)}
                    value={''}
                    style={{width:'100%'}}
                />
```
This TextInput is multiline, I use onContentSizeChange to alter the height of TextInput. 
The bug is that the value of placeholder has wrong contentSize output from onContentSizeChange.
if placeholder={'12345678'}:
output:
`{width: 0, height: 133.66666666666666}`
if placeholder={'123'}:
output:
`{width: 0, height: 50.333333333333336}`
I suppose the value of height should be width's.

This bug will make my TextInput have wrong height.

## Environment
I am using Expo
    "expo": "^26.0.0",
    "react": "16.3.0-alpha.1",
    "react-native": "https://github.com/expo/react-native/archive/sdk-26.0.0.tar.gz",(RN0.54.2)

On iOS 11.2, both simulator and real devices.
Seems no problem on Android.

## Steps to Reproduce
```
<TextInput
                    placeholder={'12345678'}
                    multiline={true}
                    onContentSizeChange={(e) => console.log(e.nativeEvent.contentSize)}
                    value={''}
                    style={{width:'100%'}}
                />
```
And change the value of placeholder to see the output. 

## Expected Behavior
The contentSize has its correct value from onContentSizeChange

## Actual Behavior
The placeholder in this project is 'Type a message...', the output of height is 240. You can clearly tell textInput is flickering when open the screen.
![giftedchat](https://user-images.githubusercontent.com/19722517/39439265-01f263b6-4c75-11e8-834d-280c8bdd80b0.gif)

**Comment by kevinscience** [comment]
Environment:
  OS: macOS Sierra 10.12.6
  Node: 9.3.0
  Yarn: 1.5.0
  npm: 5.7.1
  Watchman: Not Found
  Xcode: Xcode 9.2 Build version 9C40b
  Android Studio: 3.0 AI-171.4443003

Packages: (wanted => installed)
  react: 16.3.0-alpha.1 => 16.3.0-alpha.1
  react-native: https://github.com/expo/react-native/archive/sdk-26.0.0.tar.gz => 0.54.2

**Comment by kevinscience** [comment]
The bug is fixed after I upgrade expo to 27.0.0

**Comment by jjimenezlopez** [comment]
Hi!

I'm having the same issue and I'm not using Expo, just ReactNative through `create-react-native-app`. If I set a placeholder text for my input, the first run of `onContentSizeChange` has a height of 168 (too high) and once I enter a character, it changes to 14 (the real size).

Here is my react-native info:

```
Environment:
  OS: macOS High Sierra 10.13.4
  Node: 6.11.2
  Yarn: 1.5.1
  npm: 5.8.0
  Watchman: 4.9.0
  Xcode: Xcode 9.3 Build version 9E145
  Android Studio: 3.0 AI-171.4443003

Packages: (wanted => installed)
  react: ^16.3.0 => 16.3.2
  react-native: 0.54.1 => 0.54.1
```
